lab_4.py

```diff
@@ -0,0 +1,12 @@
+import math
+
+def func(n):
+    y = y_0 = math.sqrt(3 * n)
```

Comment:
```suggestion
    y = math.sqrt(3 * n)
```
`y_0` unnecessary variable